t/base/protocols.t

```diff
@@ -1,20 +1,23 @@
 use strict;
 use warnings;
 use Test::More;
-plan tests => 6;
 
 use LWP::UserAgent;
+plan tests => 8;
```

Comment:
From #perl-qa:

```
[11:10:31]  <@oalders>	What would be a compelling reason for not requiring a Test::More greater than 0.88? https://github.com/libwww-perl/libwww-perl/pull/88#discussion_r81676260
[11:10:33]  <+dipsy>	[ Test cleanup by genio · Pull Request #88 · libwww-perl/libwww-perl · GitHub ] 
[11:11:54]  <ilmari>	it wasn't core until 5.10.1, so you'd force 5.8 users to upgrade
[11:13:23]  <@oalders>	ah, ok.  is there any consensus on how many 5.8 users there actually are?  i see it come up every so often
[11:15:18]  <@mst>	0.88 is just for done_testing rather than no_plan
[11:15:21]  <@mst>	which is barely a big deal
```

I think it's fine to ignore my `done_testing()` suggestion. :)